substrate/frame/bounties/src/tests.rs

```diff
@@ -1481,3 +1482,326 @@ fn accept_curator_sets_update_due_correctly() {
 		);
 	});
 }
+
+#[test]
+fn poke_deposit_fails_for_insufficient_balance() {
+	ExtBuilder::default().build_and_execute(|| {
+		// Create a bounty
+		assert_ok!(Bounties::propose_bounty(RuntimeOrigin::signed(0), 50, b"12345".to_vec()));
+
+		// BountyDepositBase (80) + DataDepositPerByte (1) * description.len() (5)
+		let deposit = 85;
```

Comment:
```suggestion
		let deposit = pallet_bounty::Pallet<Test>::calculate_bounty_deposit(..)
```

Suggestion, in case the calculation changes.